infrastructure/main.tf

```diff
@@ -0,0 +1,67 @@
+terraform {
+  required_version = "~> 1.1.2"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.90"
+    }
+  }
+
+  backend "azurerm" {
+    resource_group_name  = "ioet-infra-tf-state"
+    storage_account_name = "timetrackertfstate"
+    container_name       = "time-tracker-tf-state"
+    key                  = "time-tracker-ui.tfstate"
+  }
+
+}
+
+provider "azurerm" {
+  features {}
+  skip_provider_registration = true
+}
+
+data "terraform_remote_state" "service" {
+  backend   = "azurerm"
+  workspace = terraform.workspace
+  config = {
+    resource_group_name  = "ioet-infra-tf-state"
+    storage_account_name = "timetrackertfstate"
+    container_name       = "time-tracker-tf-state"
+    key                  = "this.tfstate"
+  }
+}
+
+data "azurerm_container_registry" "registry" {
+  name                = data.terraform_remote_state.service.outputs.container_registry_name
+  resource_group_name = data.terraform_remote_state.service.outputs.resource_group_name
+}
+
+data "azurerm_resource_group" "root" {
+  name = data.terraform_remote_state.service.outputs.resource_group_name
+}
+
+locals {
+  common_name             = "time-tracker-ui"
+  environment             = terraform.workspace
+  service_name            = "${local.common_name}-${local.environment}"
+  create_app_service_plan = true
+  service_plan_kind       = "Linux"
+}
+
+module "ui" {
```

Comment:
**issue**: I don't see the image_tag definition here. That one is relevant to associate it with git releases. Take a look at https://github.com/ioet/time-tracker-v2/blob/255112b0f270ae17de84a20cd7791705dc573f8e/infrastructure/main.tf#L73 as a reference